modules/virt-connecting-to-vm-console-web.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+ifeval::["{context}" == "vnc-console"]
+:vnc-console:
+:console: VNC console
+:console-menu: VNC console
+endif::[]
+ifeval::["{context}" == "serial-console"]
+:serial-console:
+:console: serial console
+:console-menu: Serial console
+endif::[]
+ifeval::["{context}" == "desktop-viewer"]
+:desktop-viewer:
+:console: desktop viewer
+:console-menu: Desktop viewer
+endif::[]
+
+:_content-type: PROCEDURE
+[id="virt-connecting-to-vm-console-web_{context}"]
+ifdef::vnc-console,serial-console[]
+= Using the web console to connect to the {console}
+
+You can use the {product-title} web console to connect to the {console} of a virtual machine (VM).
+endif::[]
+ifdef::desktop-viewer[]
+= Using the web console to configure the {console}
+
+You can use the {product-title} web console to configure the {console} to access a Windows virtual machine (VM).
+endif::[]
+
+ifdef::vnc-console[]
+If you connect to a Windows VM with a vGPU assigned as a mediated device, you can switch between the default display and the vGPU display.
```

Comment:
Since this only applies to the specific case of a Windows VM with a vGPU assigned, I would keep it in a note block instead.